FlowCrypt/build.gradle.kts

```diff
@@ -373,12 +373,20 @@ tasks.register("renameReleaseBuilds") {
 }
 
 tasks.register<Copy>("copyReleaseApks") {
-  from("${layout.buildDirectory}") {
-    include("**/*release*.apk")
-  }
-
   includeEmptyDirs = false
-  into("${rootProject.rootDir}/release")
+
+  into(
+    File(rootProject.rootDir, "release").apply {
+      exists().takeIf { true } ?: mkdir().takeIf { true } ?: error("Can't create $name")
```

Comment:
Might be easier to read this way.
However I dont' get review notification for this PR and I think I'm not  assigned as code owner for this file.

```kt
if (!exists() && !mkdirs()) {
  error("Can't create $name")
}
```

Reply:
> Might be easier to read this way.

Agree, thank you!

> However I dont' get review notification for this PR and I think I'm not assigned as code owner for this file.

I've noticed it too. I'll improve `.github/CODEOWNERS`